docs/eventing/sources/sinkbinding/README.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "SinkBinding"
+weight: 01
+type: "docs"
+aliases:
+  - /docs/eventing/samples/sinkbinding/
+  - /docs/eventing/sources/sinkbinding
+---
+
```

Comment:
I'd go straight for removing these and adding the redirects etc to the mkdocs.yml, and adding a H1 here instead